src/pages/MarketPage/MarketPage.tsx

```diff
@@ -251,6 +253,13 @@ export const MarketPageComponent: React.FC<MarketPageProps> = ({ market }) => {
     ],
   );
 
+  const handleMintSubmission = React.useCallback(
+    async (values: FormikValues, helpers: FormikHelpers<FormikValues>) => {
+      console.log(values);
```

Comment:
well need to connect to the atomic operation here as well, perhaps this is for a different task.

Reply:
It hasn't been finished yet. Now it is complete ;)